lunes_cms/cms/serializers.py

```diff
@@ -171,6 +178,25 @@ class Meta:
         )
 
 
+class FeedbackSerializer(serializers.ModelSerializer):
+    """
+    Serializer for the Feedback module. Inherits from
+    'serializers.ModelSerializer'.
+    """
+
```

Comment:
```suggestion

    content_type = serializers.SlugRelatedField(
        slug_field="model",
        queryset=ContentType.objects.filter(
            app_label="cms", model__in=["discipline", "trainingset", "document"]
        ),
        error_messages={
            "does_not_exist": _(
                "The content type must be either 'discipline', 'training set' or 'document'."
            ),
        },
    )

    def validate(self, attrs):
        try:
            attrs["content_type"].model_class().objects.get(pk=attrs["object_id"])
        except:
            raise serializers.ValidationError(
                {
                    "object_id": [
                        _("{} with id {} does not exist.").format(
                            attrs["content_type"].name, attrs["object_id"]
                        ),
                    ]
                }
            )
        return attrs

```
I noticed two further problems:
1. When the content type is referenced by id, the app somehow has to know which ids the content types have, e.g. 1 is discipline, 2 is document and 3 is training set... instead, we should serialize the content type as string, which can be achieved by using a `SlugRelatedField`.
2. At the moment it is possible to create feedback for non-existing object, which is not what we want I think. I would add a validator to make sure this does not happen.